src/Common/src/Microsoft/Win32/SafeHandles/SafeX509Handles.Unix.cs

```diff
@@ -3,14 +3,24 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Diagnostics;
-using System.Security;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Win32.SafeHandles
 {
     internal sealed class SafeX509Handle : SafeHandle
     {
+#if DEBUG_FINALIZATIONS
+        private string _stacktrace = Environment.StackTrace;
+
+        [DllImport("libc")]
+        private static extern int printf(string f, IntPtr arg0, string arg);
```

Comment:
We have a printf Interop definition that goes through the shim:
https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Native/Interop.PrintF.cs
That doesn't have the IntPtr, but the call site could be changed instead to:
```C#
printf("%s\n\n", $"{handle:X}: {_stacktrace ?? "no stacktrace..."});
```
Could we use that instead?